src/components/Tabs/components/List/tests/List.test.tsx

```diff
@@ -0,0 +1,67 @@
+import * as React from 'react';
+import {mountWithAppProvider} from 'test-utilities';
+import List from '../List';
+import Item from '../../Item';
+
+describe('<List />', () => {
+  const mockProps = {
+    focusIndex: -1,
+    disclosureTabs: [
+      {
+        id: 'repeat-customers',
+        content: 'Repeat customers',
+      },
+      {
+        id: 'prospects',
+        content: 'Prospects',
+      },
+    ],
+  };
+
+  it('renders an unordered list', () => {
+    const list = mountWithAppProvider(<List {...mockProps} />);
+    expect(list.find('ul')).toHaveLength(1);
+  });
+
+  describe('focusIndex', () => {
+    it('passes focusIndex to correct Item', () => {
+      const focusIndex = 1;
+      const list = mountWithAppProvider(
+        <List {...mockProps} focusIndex={focusIndex} />,
+      );
+      const items = list.find(Item);
+      expect(items.first().prop('focused')).toBe(false);
+      expect(items.last().prop('focused')).toBe(true);
+    });
+  });
+
+  describe('disclosureTabs', () => {
+    it('renders a button for each item in disclosureTabs', () => {
+      const list = mountWithAppProvider(<List {...mockProps} />);
+      expect(list.find('button')).toHaveLength(2);
+    });
+
+    it('passes the id to the button', () => {
+      const list = mountWithAppProvider(<List {...mockProps} />);
```

Comment:
Same here I had to scroll up to figure out where `repeat-customers` was coming from.